ios/FluentUI/Command Bar/MultilineCommandBar.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFMultilineCommandBarRow)
+public class MultilineCommandBarRow: NSObject {
```

Comment:
Could this be a struct?

Reply:
It it is updated to a struct, it won't be available in objc. Are we ok with that?

Reply:
Probably not, let's keep it as is then.